OpenRA.Utility/Program.cs

```diff
@@ -39,6 +39,18 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
 	class Program
 	{
 		static void Main(string[] args)
+		{
+			try
+			{
+				Run(args);
+			}
+			finally
+			{
+				Log.Dispose();
```

Comment:
Its kind of crazy that the utility is writing to perf.log in the first place. Maybe not here, but it would be nice if we could adjust the logging plumbing to not be used at all for the utility.